script/Arcueid/Day 5/05_00_ARC05_1B.txt

```diff
@@ -246,11 +246,12 @@ Yeah, you've had bad luck from the get-go, haven't you?
 [sha:b8484235f9dab4fabf659b09bdc932968e72fc60]{
 -- Page 53, Offset 8722.
 -- 人を殺すにしたって、わたし以外の子にしておけばこんなコトにはならなかっただろうし」
-If you wanted to kill someone, you could have chosen someone other than me and this wouldn't have happened."
+If you wanted to kill someone, you could've chosen someone other than me, then this would've never happened."
```

Comment:
If you wanted to kill someone, you should've chosen someone other than me. This would've never happened.

Reply:
alt suggestion:
> If you wanted to kill someone, you could've chosen someone else. This whole situation could've been avoided.